src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -97,10 +98,12 @@ bool BenchmarkTableEncoder::encode(const std::string& table_name, const std::sha
     if (encoding_supports_data_type(encoding_config.default_encoding_spec.encoding_type, column_data_type)) {
       chunk_encoding_spec.push_back(encoding_config.default_encoding_spec);
     } else {
-      std::cout << " - Column '" << table_name << "." << table->column_name(column_id) << "' of type ";
-      std::cout << column_data_type << " cannot be encoded as ";
-      std::cout << encoding_config.default_encoding_spec.encoding_type << " and is ";
-      std::cout << "left Unencoded." << std::endl;
+      // Use an osyncstream to prevent multiple threads writing at the same time to std::cout.
+      auto ostream = std::basic_osyncstream(std::cout);
```

Comment:
The comment is okay, but I think using a syncstream makes the problem here already quite clear.

Further, please use `std::osyncstream` without the `basic_` prefix.